src/hammer-vlsi/defaults.yml

```diff
@@ -316,14 +316,15 @@ vlsi.inputs:
     y: 0
     # pitch (float) - pitch of bumps in microns
     pitch: 0.0
-    # cell (str) - Name of bump cell
+    # cell (str) - Name of the default bump cell
     cell: ""
     # assignments - List of BumpAssignment structs. You must specify one of name or no_connect.
     # If both are specified the bump will be left unconnected
     #  - name (Optional[str]) - The name of the net being assigned to a bump
     #  - no_connect (Optional[bool]) - If True the bump will be present but unconnected
     #  - x (int) - The X coordinate of the bump assigned to this net
```

Comment:
Based on the other PR shouldn't this mention that these can be fractions now?

Reply:
see above

Reply:
unrelated question to @colinschmidt, does our tech plugin override this default, or do we specify this at the chip level?

Reply:
This is a chip level thing. There are many reasonable cells in our tech library we are just constrained to a certain one for this project.

Reply:
You could still override it- I think it makes sense to have a sane default in the tech plugin, but at this point it would only help future projects.